book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rentable'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, person, self)
```

Comment:
- [ ] Kindly pay attention to this arrangement of the arguments. `book` comes before `person` so we should have something like this:

  ```
  Rental.new(date, self, person)
  ```

Reply:
done